browser/brave_ads/DEPS

```diff
@@ -3,4 +3,5 @@ include_rules = [
   "-bat/ledger/internal",
   "+bat/ads",
   "-bat/ads/internal",
+  "+services/service_manager/public/cpp",
```

Comment:
this should go in the most specific place possible which would be browser/brave_ads/search_result_ads/DEPS